django_ninja_crudl/controller.py

```
@@ -79,21 +78,35 @@
+        for base in bases:
+            for attr_name, attr_value in base.__dict__.items():
+                if not attr_name.startswith("__"):
+                    bases_dct[attr_name] = attr_value
+
+        # Add all attributes to dct, while allowing the child's attributes to overwrite
+        # parents' attributes
+        for attr_name, attr_value in bases_dct.items():
+            dct = bases_dct | dct
+
+        # Contruct the final API controller class
+        # bases = (APIController, ControllerBase, CrudlBaseMethodsMixin)
```

Ruff (0.8.2)

Found commented-out code

Remove commented-out code

(ERA001)